src/actions/addPackageJsonFunding.ts

```diff
@@ -0,0 +1,124 @@
+import mls from 'multilines'
+import Octokit, { ReposGetResponse } from '@octokit/rest'
+import detectIndent from 'detect-indent'
+
+import { base64 } from '../utils'
+import { getCollectiveWithGithubHandle } from '../collective'
+import { resetBranch } from '../github'
+
+import defaultFundingPackageAsString from '../assets/default-funding-package'
```

Comment:
This import seems a bit useless, can we do without and just have that as a `const` in this file?